samples/cxx/flamespeed2/flamespeed.cpp

```diff
@@ -0,0 +1,186 @@
+/*!
```

Comment:
I would much rather see an example added using the Python interface rather than the C++ interface directly. This will require adding wrappers for `enableAmbipolar` and related functions in `onedim.pyx` and `_cantera.pxd`'.